486_Project/Text_Game.py

```diff
@@ -0,0 +1,118 @@
+from Fight_Class import *
+from Characters_Class import *
+from adventurelib import *
+
+# player
+player = warrior()
+
+# enemies
+bw1 = bad_warrior()
+bw2 = bad_warrior()
+w1 = wolf()
+
+# sets state for all rooms to be empty
+Room.state = "empty"
```

Comment:
Global state-- good usage of AdventureLib here. A good alternative may also be ```Room.state = None```, which might more clearly indicate that most rooms have No/None state.